Neos.ContentRepository.Core/Classes/Feature/Common/TetheredNodeInternals.php

```diff
@@ -130,29 +130,29 @@ protected function createEventsForMissingTetheredNode(
             $childNodeAggregate = current($childNodeAggregates);
             if (!$childNodeAggregate->classification->isTethered()) {
                 throw new \RuntimeException(
-                    'We found a child node aggregate through the given node path; but it is not tethered.'
+                    'TODO: We found a child node aggregate through the given node path; but it is not tethered.'
                         . ' We do not support re-tethering yet'
-                        . ' (as this case should happen very rarely as far as we think).'
+                        . ' (as this case should happen very rarely as far as we think).',
+                    1711897665
                 );
             }
 
-            $childNodeSource = null;
-            foreach ($childNodeAggregate->getNodes() as $node) {
-                $childNodeSource = $node;
-                break;
-            }
-            /** @var Node $childNodeSource Node aggregates are never empty */
+            $occupiedDimensionSpacePoints = $childNodeAggregate->occupiedDimensionSpacePoints->getPoints();
```

Comment:
idk if we should guard here with this check ... the added tests will pass either way ;)

```php
if (count($occupiedDimensionSpacePoints) !== 1) {
    throw new \RuntimeException('TODO: The ChildNodeAggregate occupies multiple origins.', 1711897662);
}
```

Reply:
I don't see an issue with the child node aggregate occupying multiple DSPs, that's totally legit